python/tvm/relay/frontend/onnx.py

```diff
@@ -770,10 +770,8 @@ def flatten_to_nd(x, x_shape, nd=3):
                 # Convert a and b into 3 dimensional tensors.
                 a = flatten_to_nd(inputs[0], a_shape, 3)
                 b = flatten_to_nd(inputs[1], b_shape, 3)
-                # Transpose matrix dimensions of b.
-                b = _op.transpose(b, [0, 2, 1])
                 # Perform a batch matmul.
-                output = _op.nn.batch_matmul(a, b)
+                output = _op.nn.batch_matmul(a, b, transpose_b=False)
```

Comment:
Just a little concern about changing the default behavior of framework frontend since currently the default topi schedule support for NN format is not as strong as the original NT one.
This may cause confusions to those who have used onnx frontend before.

To give an example, I've added an extra config to TensorFlow frontend which uses the NT format by default but provides a option to use normal format.

Reply:
@jcf94 Thanks for the pointer. I will refactor to make NN optional.